exampleSite/content/docs/guide/shortcodes/_index.md

```diff
@@ -17,6 +17,7 @@ Hextra provides a collection of beautiful shortcodes to enhance your content.
   {{< card link="icon" title="Icon" icon="badge-check" >}}
   {{< card link="steps" title="Steps" icon="one" >}}
   {{< card link="tabs" title="Tabs" icon="collection" >}}
+  {{< card link="asciinema" title="Asciinema Player" icon="terminal" >}}
```

Comment:
for now let's move it to the section below under "Additional shortcodes provided by Hugo and Hextra:"